src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -2631,10 +2632,24 @@ func (config *Config) appendFDs(fds []*os.File) []int {
 	return fdInts
 }
 
-func (config *Config) appendSeccompSandbox() {
+func (config *Config) appendSeccompSandbox(logger QMPLog) {
 	if config.SeccompSandbox != "" {
 		config.qemuParams = append(config.qemuParams, "-sandbox")
 		config.qemuParams = append(config.qemuParams, config.SeccompSandbox)
+		out, err := pkgUtils.RunCommand([]string{"cat", "/proc/sys/net/core/bpf_jit_enable"})
```

Comment:
For ease of testing, I wonder if we should make a function to check if bpf_jit_enable is set?

Reply:
Also -- this check should be at virtcontainers layer  -- we need to avoid adding a katautils dependency inside govmm (govmm shouldn't import any kata pkgs)